lib/common/subredditPost.dart

```diff
@@ -49,20 +56,20 @@ class SubredditPost extends StatelessWidget {
                         crossAxisAlignment: CrossAxisAlignment.start,
                         children: [
                           Text(
-                            submissionData.title,
+                            widget.submissionData.title,
                             style: GoogleFonts.poppins(
                               fontWeight: FontWeight.w400,
                             ),
                           ),
                           Text(
-                            submissionData.author,
+                            widget.submissionData.author,
                             style: GoogleFonts.poppins(
                               color: Colors.blueGrey,
                               fontSize: 11,
                             ),
                           ),
                           Text(
-                            '${NumberFormat.compact().format(submissionData.upvotes)} upvotes  •  ${submissionData.numComments.toString()} comments  •  ${submissionData.subreddit.displayName}',
+                            '${NumberFormat.compact().format(widget.submissionData.upvotes)} upvotes  •  ${widget.submissionData.numComments.toString()} comments  •  ${widget.submissionData.subreddit.displayName}',
```

Comment:
Should we add some indicator for when the post is upvoted / downvoted?

`${NumberFormat.compact().format(widget.submissionData.upvotes)}`

I think at a minimum we should add `1` to the upvotes value or subtract `1` to the upvotes value. Also, maybe even changing the colour of the text to the `upVoted` colour.

Reply:
I have just added the +1 and -1 feature. I already implemented the colour change of the upvote/downvote button. If I click on the upvote/downvote button it will change its colour to red.